test/test_parse_en.py

```diff
@@ -687,6 +687,34 @@ def testExtract(text, expected_date, expected_leftover):
         testExtract("what's the weather like next tuesday night",
                     "2017-07-04 22:00:00", "what is weather like night")
 
+    def test_extract_ambiguous_month_en(self):
+        dec = datetime(2017, 12, 27, 8, 1, 2)
+        jun = datetime(2017, 6, 27, 20, 1, 2)
+        self.assertEqual(
+            extract_datetime('when is september', jun)[0],
+            datetime(2017, 9, 1, tzinfo=default_timezone()))
+        self.assertEqual(
+            extract_datetime('when was september', dec)[0],
+            datetime(2017, 9, 1, tzinfo=default_timezone()))
```

Comment:
Should this always return the first of the month, or should "was" go for the last day `(2017, 9, 30)`?

Reply:
when we had to make this sort of decision we usually have been choosing the start of the date range.

As discussed in one of the other PRs/Issues i think what we really need is extracting date ranges with error confidences